docs/css/footer.css

```diff
@@ -0,0 +1,23 @@
+#footer{
```

Comment:
id selector in CSS works but I think I read somewhere that it shouldn't be used in CSS. `id` should only be used with javascript. (not mandatory but Best Practice)